src/controllers/executionPlanWebviewController.ts

```diff
@@ -118,49 +114,54 @@ export class ExecutionPlanWebviewController extends ReactWebviewPanelController<
                 language: "xml",
             });
 
-            await vscode.window.showTextDocument(planXmlDoc);
+            void vscode.window.showTextDocument(planXmlDoc);
 
             return state;
         });
         this.registerReducer("showQuery", async (state, payload) => {
-            await this.untitledSqlDocumentService.newQuery(payload.query);
+            void this.untitledSqlDocumentService.newQuery(payload.query);
 
             return state;
         });
         this.registerReducer("updateTotalCost", async (state, payload) => {
-            this.state.totalCost += payload.totalCost;
+            this.state.executionPlanState.totalCost += payload.addedCost;
 
             return {
                 ...state,
-                totalCost: this.state.totalCost,
+                executionPlanState: {
+                    ...state.executionPlanState,
+                    totalCost: this.state.executionPlanState.totalCost,
+                },
             };
         });
     }
 
-    private async createExecutionPlanGraphs() {
-        if (!this.state.executionPlan) {
-            const planFile: ep.ExecutionPlanGraphInfo = {
-                graphFileContent: this.executionPlanContents,
-                graphFileType: ".sqlplan",
-            };
-            try {
-                this.state.executionPlan =
-                    await this.executionPlanService.getExecutionPlan(planFile);
-                this.state.executionPlanGraphs =
-                    this.state.executionPlan.graphs;
-                this.state.loadState = ApiStatus.Loaded;
-                this.state.totalCost = this.calculateTotalCost();
-            } catch (e) {
-                this.state.loadState = ApiStatus.Error;
-                this.state.errorMessage = e.toString();
-            }
+    private async createExecutionPlanGraphs(content: string) {
+        const planFile: ep.ExecutionPlanGraphInfo = {
+            graphFileContent: content,
+            graphFileType: ".sqlplan",
```

Comment:
Can you move the file extension into a constants file and swap all the literal strings?